### PR TITLE
Big VS Small component benchmarks.

### PR DESCRIPTION
**Pull request by jojolepro**
## Checklist

* [X] I've added tests for all code changes and additions (where applicable)

## API changes
N/A

---
This change is [Reviewable](https://reviewable.io/reviews/slide-rs/specs/583)

### CODE REVIEW

File: src/world/entity.rs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 use std::{
     fmt,
-    num::NonZeroI32,
     sync::atomic::{AtomicUsize, Ordering},
 };
 
+use nonzero_signed::NonZeroI32;
```
Comment (torkleyy) [inline src/world/entity.rs]: I'm confused by this; it's now deprecated since std has support for it.

Comment (jojolepro) [reply src/world/entity.rs]: the import was failing for me

Comment (torkleyy) [reply src/world/entity.rs]: Have you updates your Rust toolchain?

Comment (jojolepro) [reply src/world/entity.rs]: fixed thanks

**Pull request merged**